api/oc_push.c

```diff
@@ -254,10 +255,6 @@ set_ns_properties(oc_resource_t *resource, oc_rep_t *rep, void *data)
   (void)resource;
   bool pushtarget_is_updated = false;
 
-#define IS_PROPERTY(rep, prop)                                                 \
```

Comment:
It will be much pretty that you call:
`oc_rep_is_property(rep, prop, OC_CHAR_ARRAY_LEN(prop)`